addons/skin.estuary/xml/DialogVideoVersion.xml

```diff
@@ -1,167 +1,399 @@
+				<shadowcolor>black</shadowcolor>
+			</control>			
+			<control type="group">
+				<visible>String.IsEqual(window(home).Property(dialogvideoversion),extras)</visible>
+				<include content="DialogBackgroundCommons">
+					<param name="width" value="1640" />
+					<param name="height" value="780" />
+					<param name="header_label" value="$LOCALIZE[31617]$INFO[ListItem.Title,: ]$INFO[ListItem.Year, (,)]" />
+					<param name="header_id" value="0" />
+				</include>
+			</control>	
+			<control type="image">
+				<left>20</left>
+				<width>400</width>
+				<height>630</height>
+				<top>100</top>
+				<texture>$VAR[VideoVersionPosterVar]</texture>
+				<aspectratio aligny="top">keep</aspectratio>
 			</control>
-			<control type="group" id="100">
-				<top>90</top>
-				<left>320</left>
-				<control type="image">
-					<top>0</top>
-					<left>0</left>
-					<width>680</width>
-					<height>600</height>
-					<texture border="21">dialogs/dialog-bg.png</texture>
-				</control>
-				<control type="list" id="50">
-					<description>version list</description>
-					<visible>ControlGroup(200).HasFocus(201)</visible>
-					<top>20</top>
-					<left>20</left>
-					<height>580</height>
-					<onup>50</onup>
-					<ondown>50</ondown>
-					<onleft>200</onleft>
-					<onright>60</onright>
-					<pagecontrol>60</pagecontrol>
-					<scrolltime>200</scrolltime>
-					<include content="DefaultSimpleListLayout">
-						<param name="width" value="640" />
-						<param name="height" value="80" />
-						<param name="list_id" value="50" />
-						<param name="align" value="left" />
-					</include>
-				</control>
-				<control type="list" id="51">
-					<description>extras list</description>
-					<visible>ControlGroup(200).HasFocus(202)</visible>
-					<top>20</top>
-					<left>20</left>
-					<onup>51</onup>
-					<ondown>51</ondown>
-					<height>580</height>
-					<onleft>200</onleft>
-					<onright>60</onright>
-					<pagecontrol>60</pagecontrol>
-					<scrolltime>200</scrolltime>
-					<include content="DefaultSimpleListLayout">
-						<param name="width" value="640" />
-						<param name="height" value="80" />
-						<param name="list_id" value="51" />
-						<param name="align" value="left" />
-					</include>
-				</control>
-				<control type="scrollbar" id="60">
-					<left>660</left>
-					<top>20</top>
-					<width>12</width>
-					<height>560</height>
-					<onleft condition="Control.IsVisible(50)">50</onleft>
-					<onleft condition="Control.IsVisible(51)">51</onleft>
-					<onright condition="Window.IsActive(videoversion)">101</onright>
-					<orientation>vertical</orientation>
-				</control>
+			<control type="label">
+				<description>Path label</description>
+				<left>20</left>
+				<bottom>10</bottom>
+				<width>1300</width>
+				<height>30</height>
+				<font>font12</font>
+				<align>right</align>
+				<aligny>bottom</aligny>
+				<textcolor>grey</textcolor>
+				<label>$INFO[ListItem.FileNameAndPath]</label>
```

Comment:
Use a $VAR here to get `Container(ID).ListItem.FileNameAndPath` for active list (see @CrystalP comments)

i.e.
```
<variable name="VideoVersionsFileNameAndPath">
    <value condition="String.IsEqual(window(home).Property(dialogvideoversion),extras)">$INFO[Container(51).ListItem.FileNameAndPath]</value>
    <value>$INFO[Container(50).ListItem.FileNameAndPath]</value>
</variable>
```